.github/workflows/webpage.yml

```diff
@@ -26,53 +21,36 @@ jobs:
       COMMIT_AUTHOR: Deploy Action
       COMMIT_AUTHOR_EMAIL: action@github.com
     steps:
-    - uses: actions/checkout@v2
-    - run: git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-    - name: Install prerequisites
-      run: pip install --upgrade wheel setuptools
-    - name: Install
-      run: pip install -e . -v
-    - name: Get cloned data from gh-pages
-      run: |
-        git remote set-url origin https://github.com/aiidateam/aiida-registry.git
-        git fetch origin gh-pages
-        git checkout origin/gh-pages plugins_metadata.json || true
-        mv plugins_metadata.json cloned_plugins_metadata.json || true
-
-    - name: Restore cached Plugins
-      id: cache-plugins-restore
-      uses: actions/cache/restore@v3
-      with:
-        path: |
-          plugins_metadata.json
-        key: plugins_metadata
+    - name: Checkout Repo ⚡️
+      uses: actions/checkout@v3
+    - name: Create dev environment
+      uses: ./.github/actions/create-dev-env
 
     - name: fetch metadata
       if: steps.cache-plugins-restore.outputs.cache-hit != 'true'
       id: fetch_metadata
       env:
+        # Use the GITHUB_TOKEN for github API calls
+        # otherwise the rate limit will be exceeded
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: aiida-registry fetch
-
-    - name: Get entry points data
+    - name: Check plugins installation
+      # This step will attach plugin installation inforamtion to the metadata, e.g. if the plugin can be installed or not
       run: aiida-registry test-install
-    - name: Move the JSON file to the React directory
+
+    - name: Move JSON file to the React project
       run: |
-        mv plugins_metadata.json aiida-registry-app/src/
+        cp plugins_metadata.json aiida-registry-app/src/
+        cp plugins_metadata.json aiida-registry-app/dist/
```

Comment:
`npm run build` does not do that?

Reply:
> npm run build does not already do that?

Yes it does. I think running `mv` command only will do the job. What you think @unkcpz ?

Reply:
The second copy is for the code at https://github.com/aiidateam/aiida-registry/blob/701a0d96ad88c7744554bb84b2992d72d849f878/.github/workflows/preview.yml#L54, was that needed?

Reply:
In `preview.yml`, the file will not be on `gh-pages` without this step. However, it is not needed on `webpage.yml`. Not sure why🤷‍♂️